src/content/guides/farmlands-card-partner-support.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Farmlands Card Partner Support Guide
+img: /farmlands-pos-integration-cover.jpg
+description: How to for Farmlands Card Partners and Card Holders to request support from the relevant company and teams to get the help they need.
```

Comment:
I think there's an opportunity to make this shorter. Longer SEO descriptions will be truncated on some sites so it's nice to make as much visible to the reader as possible. Ultimately, we want them to click the link 💪 

```suggestion
description: How to for Farmlands Card Partners and Card Holders to get the help they need.
```